### PR TITLE
Fix Apache kafka and Apache httpd importer

### LINKED ISSUE #1174: Failed to run Apache Kafka importer
```
Importing data using vulnerabilities.importers.apache_kafka.ApacheKafkaImporter
Traceback (most recent call last):
  File "/home/tg1999/Desktop/vulnerablecode/vulnerabilities/management/commands/import.py", line 60, in import_data
    ImportRunner(importer).run()
  File "/home/tg1999/Desktop/vulnerablecode/vulnerabilities/import_runner.py", line 43, in run
    advisory_datas = importer_class().advisory_data()
  File "/home/tg1999/Desktop/vulnerablecode/vulnerabilities/importers/apache_kafka.py", line 103, in advisory_data
    parsed_data = self.to_advisory(advisory_page)
  File "/home/tg1999/Desktop/vulnerablecode/vulnerabilities/importers/apache_kafka.py", line 140, in to_advisory
    cve_version_mapping = affected_version_range_mapping[cve_id]
KeyError: 'CVE-2023-25194'
Failed to run importer vulnerabilities.importers.apache_kafka.ApacheKafkaImporter. Continuing...
CommandError: 1 failed!: vulnerabilities.importers.apache_kafka.ApacheKafkaImporter
```

### PR DESCRIPTION
**Pull request by TG1999**
Fixes #1174 and #1175

### CODE REVIEW

File: vulnerabilities/importers/apache_kafka.py
Comment (johnmhoran) [inline vulnerabilities/importers/apache_kafka.py:1]: This looks good @TG1999 .  One thought: we should probably change the comment references on line 142 and line 148 from triggering/throwing a `KeyError` to your new (superior) use of `logger.error()`.

Comment (TG1999) [reply vulnerabilities/importers/apache_kafka.py:1]: Makes sense @johnmhoran , thanks for pointing it out.

File: vulnerabilities/importers/apache_kafka.py [hunk 1]
```diff
@@ -135,13 +139,18 @@ def to_advisory(self, advisory_page):
...
 
-            # This throws a KeyError if the opening h2 tag `id` data changes or is not in the
-            # hard-coded affected_version_range_mapping dictionary.
-            cve_version_mapping = affected_version_range_mapping[cve_id]
-            if cve_version_mapping["action"] == "include":
+            cve_version_mapping = affected_version_range_mapping.get(cve_id)
+            if not cve_version_mapping:
+                logger.error(f"Data for {cve_id} not found in mapping. Skipping.")
+            if cve_version_mapping and cve_version_mapping.get("action") == "include":
                 # These 2 variables (not used elsewhere) trigger the KeyError for changed/missing data.
```
Comment (TG1999) [inline vulnerabilities/importers/apache_kafka.py:146]: Remove this comment.

**Review by pombredanne — APPROVED** [review]
LGTM!

**Pull request merged**